view/src/components/produce.js

```diff
@@ -1,617 +1,723 @@
+                      label="Shipping Maintenance Temperature High"
+                      name="shippingMaintenanceTemperatureHigh"
+                      type="number"
+                      autoComplete="shippingMaintenanceTemperatureHigh"
+                      InputProps={{
+                        endAdornment: (
+                          <InputAdornment position="end">°F</InputAdornment>
+                        ),
+                      }}
+                      helperText={errors.shippingMaintenanceTemperatureHigh}
+                      value={this.state.shippingMaintenanceTemperatureHigh}
+                      error={
+                        errors.shippingMaintenanceTemperatureHigh ? true : false
+                      }
+                      onChange={this.handleChange}
+                    />
+                  </Grid>
+                  <Grid item xs={4}>
+                    <TextField
+                      variant="outlined"
+                      required
+                      fullWidth
+                      id="shippingPresetTemperature"
+                      label="Shipping Preset Temperature"
+                      name="shippingPresetTemperature"
+                      type="number"
+                      autoComplete="shippingPresetTemperature"
+                      InputProps={{
+                        endAdornment: (
+                          <InputAdornment position="end">°F</InputAdornment>
+                        ),
+                      }}
+                      helperText={errors.shippingPresetTemperature}
+                      value={this.state.shippingPresetTemperature}
+                      error={errors.shippingPresetTemperature ? true : false}
+                      onChange={this.handleChange}
+                    />
+                  </Grid>
+                </Grid>
+              </form>
+            </Container>
+          </Dialog>
+          <Container maxWidth="lg">
+            <Grid container spacing={2} alignItem="center">
+              <Grid item xs={12}>
+                <div>
+                  <input
+                    type="text"
+                    value={value}
+                    onChange={this.handleSearch}
+                  />
+                </div>
+
+                <div className={classes.search}>
+                  <div className={classes.searchIcon}>
+                    <SearchIcon />
+                  </div>
+                  <InputBase
+                    fullWidth={true}
+                    placeholder="Search…"
+                    classes={{
+                      root: classes.inputRoot,
+                      input: classes.inputInput,
+                    }}
+                    inputProps={{ "aria-label": "search" }}
+                    value={this.state.value}
+                    onChange={this.handleSearch}
+                  />
+                </div>
+              </Grid>
+              <SearchResults
```

Comment:
Did you figure out why the spacing between the cards changed when moved into the SearchResults component?

Reply:
still working on this - it might become clearer when i work on the other files or when i merge with the new master, so i'll come back to this afterwards. no clear changes in the styles section, and using keys in the arrays didn't adjust this as well

Reply:
fixed by creating a new grid inside the render results function